components/Layout.js

```diff
@@ -59,6 +76,8 @@ const MainContent = styled.main`
   padding: 1rem;
   padding-top: 0;
   margin-bottom: 65px;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  _background-color: #efefef;
```

Comment:
If you don't need a property, I would just remove it.

Reply:
Just removed! :)